sql/core/pom.xml

```diff
@@ -130,6 +130,12 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>org.apache.avro</groupId>
```

Comment:
Thank you for review, @srowen and @robbinspg .
* Yes. This is the only module. 
* 1.8.0 cames from Parquet [here](https://github.com/apache/parquet-mr/blob/parquet-1.8.x/pom.xml#L96).
* Top-level `avro.version` is synced with Hadoop avro version. Here, `apache.parquet.avro.AvroParquetWriter` in the testcode only requires that.